### PR TITLE
easyeda2kicad: init at 0.8.0

### PR DESCRIPTION
**Pull request by ChocolateLoverRaj**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -3816,6 +3816,13 @@
     githubId = 98980;
     name = "Chmouel Boudjnah";
   };
+  ChocolateLoverRaj = {
+    email = "paranjperajas@gmail.com";
+    github = "ChocolateLoverRaj";
+    githubId = 52586855;
+    matrix = "@chocolateloverraj:matrix.org";
+    name = "Rajas Paranjpe";
+  };
```
Comment (oddlama) [inline maintainers/maintainer-list.nix:3825]: Please put this into a separate commit before your main commit with the commit message `maintainers: add ChocolateLoverRaj`

File: pkgs/by-name/ea/easyeda2kicad/package.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  dependencies = with python3Packages; [
+    setuptools
+    pydantic
+    requests
+  ];
+
+  meta = with lib; {
+    description = "Convert any LCSC components (including EasyEDA) to KiCad library";
+    homepage = "https://github.com/uPesy/easyeda2kicad.py";
```
Comment (oddlama) [inline pkgs/by-name/ea/easyeda2kicad/package.nix:24]: Please also add `mainProgram` to the meta section

**Comment by oddlama** [comment]
Please ensure that all CI check pass successfully, it looks like your maintainer list entry is not sorted correctly and the main package file has not been formatted using `nixfmt-rfc-style`

File: pkgs/by-name/ea/easyeda2kicad/package.nix [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  python3Packages,
+  pythonPackages,
```
Comment (oddlama) [inline pkgs/by-name/ea/easyeda2kicad/package.nix:3]: You should probably refer to `fetchPypi` by directly including it here in the inputs, and remove `pythonPackages` which is you don't need (it's for python2).

File: maintainers/maintainer-list.nix [hunk 4]
```diff
@@ -941,11 +941,7 @@
     githubId = 16022854;
     matrix = "@albertodvp:matrix.org";
     name = "Alberto Fanton";
-    keys = [
-      {
-        fingerprint = "63FD 3A4F 4832 946C B808  8E3C C852 4052 69E7 A087";
-      }
-    ];
+    keys = [ { fingerprint = "63FD 3A4F 4832 946C B808  8E3C C852 4052 69E7 A087"; } ];
```
Comment (bjornfor) [inline maintainers/maintainer-list.nix:944]: Please undo these unrelated formatting changes.

**Review by bjornfor — APPROVED** [review]
